pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1516,3 +1516,35 @@ def test_hypothesis_delimited_date(date_format, dayfirst, delimiter, test_dateti
 
     assert except_out_dateutil == except_in_dateutil
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "names, usecols, parse_dates, missing_cols",
+    [
+        (None, ["val"], ["date", "time"], "date, time"),
```

Comment:
add a tuple or other list-like in this test

Reply:
When I use tuple, [_validate_parse_dates_arg](https://github.com/pandas-dev/pandas/blob/master/pandas/io/parsers.py#L1314) throws the following error.
> TypeError: Only booleans, lists, and dictionaries are accepted for the 'parse_dates' parameter

Reply:
Yea I guess this is only currently documented as supporting scalars, lists and dicts (not tuples)